Sources/Plasma/PubUtilLib/plPipeline/CMakeLists.txt

```diff
@@ -71,6 +71,7 @@ target_link_libraries(plPipeline
         plSurface
         pfCamera # plCaptureRender, plDynamicEnvMap, plDXPipeline
         pfGameGUIMgr # plCaptureRender
+        freetype
```

Comment:
Out of curiosity, should this also have a (platform-specific) link for `CoreText.framework`?

Reply:
CoreText is under the Cocoa/Cocoa.h umbrella, so it should get loaded under Cocoa. Cocoa is kind of a catch all for a bunch of stuff.

Reply:
We need to (conditionally on Windows, I guess) add `plWinDpi` as a private target link library here to fix build issues with the unit tests and 3DS Max plugin